### ISSUE TITLE
Anti-Enterocyte Antibody

### ISSUE THREAD

**Issue by MaudDLVT** [obophenotype/human-phenotype-ontology#7234]
**Preferred term label:**
Anti-Enterocyte antibody positivity

**Synonyms**
AEA

**Definition (free text, please give PubMed ID)**
PMID: 33744411, PMID: 24051695

**Parent term (use hpo.jax.org/app)**
HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:522037

**Your nano-attribution (ORCID)**
0000-0002-3387-1836

**Comment by pnrobinson** [comment]
added
